File: LeetCode/70. Climbing Stairs.swift
class Solution {
func climbStairs(_ n: Int) -> Int {
    var arr = [Int](repeating: 0, count: n+1)
    arr[0] = 1
    arr[1] = 2
    if n > 2 {
        let result = 0
        for i in 2...n-1 {
            arr[i] = arr[i-1] + arr[i-2]
        }
    }
    return arr[n-1]
}
}

File: LeetCode/70. Climbing Stairs.py
from collections import deque
class Solution:
    def climbStairs(self, n: int) -> int:
        arr = deque([0, 1, 2])
        if n < 3: return arr[n]
        for _ in range(3, n + 1):
            arr.append(arr[-1] + arr[-2])
            arr.popleft()
        return arr[-1]
